tests/test_carousel.py

```diff
@@ -0,0 +1,41 @@
+from dash import Dash, html, Output, Input, _dash_renderer
+import dash_mantine_components as dmc
+
+_dash_renderer._set_react_version("18.2.0")
+
+def test_001po_popover(dash_duo):
+    app = Dash(__name__, external_stylesheets=dmc.styles.ALL)
+
+    component = dmc.Carousel(
+        [
+            dmc.CarouselSlide(dmc.Center("Slide-0", bg="blue", c="white", p=60)),
+            dmc.CarouselSlide(dmc.Center("Slide-1", bg="blue", c="white", p=60)),
+            dmc.CarouselSlide(dmc.Center("Slide-2", bg="blue", c="white", p=60)),
+        ],
+        id="carousel",
+        autoplay={"delay": 500, "stopOnMouseEnter": True, "stopOnInteraction": False, "stopOnLastSnap": True},
```

Comment:
maybe disable `stopOnMouseEnter` just so people don't accidentally halt the test when running it locally?